settings/installer/ospd/ospd.spec

```diff
@@ -12,14 +12,27 @@ subparsers:
 
             - title: Product
               options:
-                product:
-                    type: YamlFile
-                    help: The product to install
+                product-version:
+                    type: Value
+                    help: The product version
                     required: yes
-                product-repo:
-                    type: YamlFile
-                    help: The product repo type
-                    default: puddle.yml
+                    choices: ["7", "8"]
+                product-build:
+                    type: Value
+                    help: The product build
+                    default: latest
+                product-extra:
+                    type: Value
+                    help: The product extra info
+                product-core-version:
+                    type: Value
+                    help: The product core version
+                product-core-build:
+                    type: Value
+                    help: The product core build
+                product-core-extra:
```

Comment:
no need for the "extra", the place holder is just until we support poodles which requires an "extra" flag.
even without this flag, a user can do -e installer.product.extra="-d -r 7.1" etc